Makefile

```diff
@@ -215,20 +138,19 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
```

Comment:
Given controller-gen is no longer a target, it needs to be removed from here too.
```bash
.PHONY: manifests
manifests: ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
```